### PR TITLE
Update to video.js 5.x

### PR DESCRIPTION
**Pull request by dmlap**
Add a duration setter to MediaSources.

### CODE REVIEW

File: dash/dash-eme.js [hunk 1]
```diff
@@ -1,213 +0,0 @@
-function prefixedAttributeName(obj, suffix, opt_preprefix) {
```
Comment (dmlap) [inline dash/dash-eme.js]: I removed the experimental DASH player we used in this project long ago to test for native Media Source compatibility. We'll be converting contrib-hls to use real media sources soon which will be a much more robust compatibility check.

File: src/videojs-media-sources.js [hunk 2]
```diff
@@ -262,31 +279,4 @@
     }
   };
 
-  // plugin
-  videojs.plugin('mediaSource', function(options){
```
Comment (dmlap) [inline src/videojs-media-sources.js]: This was unused as far as I'm aware.

File: src/videojs-media-sources.js [hunk 3]
```diff
@@ -100,6 +100,23 @@
...
     return sourceBuffer;
   };
+
+  /**
+   * Set or return the presentation duration.
+   * @param value {double} the duration of the media in seconds
+   * @param {double} the current presentation duration
+   * @see http://www.w3.org/TR/media-source/#widl-MediaSource-duration
+   */
+  videojs.MediaSource.prototype.duration = function(value){
```
Comment (dmlap) [inline src/videojs-media-sources.js]: Here's the only enhancement in this PR.

**Comment by imbcmdth** [comment]
LGTM :pisces:

**Comment by dmlap** [comment]
Obsoleted by #41.

**Pull request closed without merging**